<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientWindow</class>
 <widget class="QMainWindow" name="ClientWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>653</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chat Client</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="cltConfigDisplay">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>271</width>
      <height>81</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="cltChatDisplay">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>611</width>
      <height>261</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="MessageBox" name="cltChatEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>611</width>
      <height>71</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="cltSendButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>450</y>
      <width>90</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="cltSaveButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Chat Session</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>653</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MessageBox</class>
   <extends>QTextEdit</extends>
   <header>messagebox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
